command/operator_diagnose_test.go

```diff
@@ -41,130 +41,130 @@ func TestOperatorDiagnoseCommand_Run(t *testing.T) {
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "test-consul-direct-access-storage",
+							Name:   "Consul Storage Direct Server Access",
 							Status: diagnose.OkStatus,
 						},
 					},
 				},
 				{
-					Name:   "service-discovery",
+					Name:   "Service Discovery",
 					Status: diagnose.OkStatus,
 					Children: []*diagnose.Result{
 						{
-							Name:   "test-serviceregistration-tls-consul",
+							Name:   "Consul Service Discovery TLS Checks",
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "test-consul-direct-access-service-discovery",
+							Name:   "Consul Service Discovery Direct Server Access Checks",
 							Status: diagnose.OkStatus,
 						},
 					},
 				},
 				{
-					Name:   "create-seal",
+					Name:   "Create Vault Server Configuration Seals",
 					Status: diagnose.OkStatus,
 				},
 				{
-					Name:   "setup-core",
+					Name:   "Create Core Configuration",
 					Status: diagnose.OkStatus,
 					Children: []*diagnose.Result{
 						{
-							Name:   "init-randreader",
+							Name:   "Initialize Randomness for Core",
 							Status: diagnose.OkStatus,
 						},
 					},
 				},
 				{
-					Name:   "setup-ha-storage",
+					Name:   "HA Storage",
 					Status: diagnose.OkStatus,
 					Children: []*diagnose.Result{
 						{
-							Name:   "create-ha-storage-backend",
+							Name:   "Create HA Storage Backend",
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "test-consul-direct-access-storage",
+							Name:   "Consul HA Storage Direct Server Access",
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "test-ha-storage-tls-consul",
+							Name:   "Consul TLS Checks",
 							Status: diagnose.OkStatus,
 						},
 					},
 				},
 				{
-					Name:   "determine-redirect",
+					Name:   "Determine Redirect Address",
 					Status: diagnose.OkStatus,
 				},
 				{
-					Name:   "find-cluster-addr",
+					Name:   "Cluster Address Checks",
 					Status: diagnose.OkStatus,
 				},
 				{
-					Name:   "init-core",
+					Name:   "Core Creation Checks",
 					Status: diagnose.OkStatus,
 				},
 				{
-					Name:   "init-listeners",
+					Name:   "Start Listeners",
 					Status: diagnose.WarningStatus,
 					Children: []*diagnose.Result{
 						{
-							Name:   "create-listeners",
+							Name:   "Create Listeners",
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "check-listener-tls",
+							Name:   "Listener TLS Checks",
 							Status: diagnose.WarningStatus,
 							Warnings: []string{
-								"TLS is disabled in a Listener config stanza.",
+								"TLS is disabled in a listener config stanza.",
 							},
 						},
 					},
 				},
 				{
-					Name:    "unseal",
+					Name:    "Barrier Encryption Checks",
 					Status:  diagnose.ErrorStatus,
 					Message: "Diagnose could not create a barrier seal object",
 				},
 				{
-					Name:   "start-servers",
+					Name:   "Server Runtime Checks",
```

Comment:
Noun to verb suggestion: "Check Server Runtime"

Reply:
Changed to "Check Server Before Runtime" just to clarify.